.github/workflows/pullrequest.yml

```diff
@@ -87,7 +87,7 @@ jobs:
       - name: Calculate cache keys for generated files
         id: calculate_generated_files_cache_key
         run: |
-          export HASH=${{ hashFiles('scripts/codegen.js', 'libs/cms/src/lib/generated/contentfulTypes.d.ts', 'apps/air-discount-scheme/web/i18n/withLocale.tsx', 'apps/air-discount-scheme/web/components/AppLayout/AppLayout.tsx', 'apps/air-discount-scheme/web/components/Header/Header.tsx', 'apps/air-discount-scheme/web/screens/**.tsx', 'libs/application/types/src/lib/ApplicationTypes.ts', 'apps/**/codegen.yml', 'libs/**/codegen.yml', 'apps/**/*.model.ts', 'libs/**/*.model.ts', 'apps/**/*.enum.ts', 'libs/**/*.enum.ts', 'apps/**/queries/**/*.tsx?', 'libs/**/queries/**/*.tsx?', 'libs/**/mutations/**/*.tsx?', 'libs/**/fragments/**/*.tsx?' , 'apps/**/*.resolver.ts', 'libs/**/*.resolver.ts', 'apps/**/*.service.ts', 'libs/**/*.service.ts', 'apps/**/*.dto.ts', 'libs/**/*.dto.ts', 'apps/**/*.input.ts', 'libs/**/*.input.ts', 'apps/**/*.module.ts', 'libs/**/*.module.ts', 'apps/**/*.controller.ts', 'libs/**/*.controller.ts', 'apps/**/*.union.ts', 'libs/**/*.union.ts', 'apps/**/*.graphql.ts', 'apps/judicial-system/**/*.graphql', 'libs/**/*.graphql.ts', 'libs/**/*.graphql', 'libs/**/clientConfig.yaml') }}
+          export HASH=${{ hashFiles('scripts/codegen.js', 'libs/cms/src/lib/generated/contentfulTypes.d.ts', 'apps/air-discount-scheme/web/i18n/withLocale.tsx', 'apps/air-discount-scheme/web/components/AppLayout/AppLayout.tsx', 'apps/air-discount-scheme/web/components/Header/Header.tsx', 'apps/air-discount-scheme/web/screens/**.tsx', 'libs/application/types/src/lib/ApplicationTypes.ts', 'apps/**/codegen.*', 'libs/**/codegen.*', 'apps/**/*.model.ts', 'libs/**/*.model.ts', 'apps/**/*.enum.ts', 'libs/**/*.enum.ts', 'apps/**/queries/**/*.tsx?', 'libs/**/queries/**/*.tsx?', 'libs/**/mutations/**/*.tsx?', 'libs/**/fragments/**/*.tsx?' , 'apps/**/*.resolver.ts', 'libs/**/*.resolver.ts', 'apps/**/*.service.ts', 'libs/**/*.service.ts', 'apps/**/*.dto.ts', 'libs/**/*.dto.ts', 'apps/**/*.input.ts', 'libs/**/*.input.ts', 'apps/**/*.module.ts', 'libs/**/*.module.ts', 'apps/**/*.controller.ts', 'libs/**/*.controller.ts', 'apps/**/*.union.ts', 'libs/**/*.union.ts', 'apps/**/*.graphql.ts', 'apps/judicial-system/**/*.graphql', 'libs/**/*.graphql.ts', 'libs/**/*.graphql', 'libs/**/clientConfig.*') }}
```

Comment:
perhaps break up the line for future extensions so we can see diffs more clearly?

Not sure if `hashFiles()` supports multiline in a bash context but we could test it.
```suggestion
          export HASH=${{ hashFiles(
            'scripts/codegen.js',
            'libs/cms/src/lib/generated/contentfulTypes.d.ts',
            'apps/air-discount-scheme/web/i18n/withLocale.tsx',
            'apps/air-discount-scheme/web/components/AppLayout/AppLayout.tsx',
            'apps/air-discount-scheme/web/components/Header/Header.tsx',
            'apps/air-discount-scheme/web/screens/**.tsx',
            'libs/application/types/src/lib/ApplicationTypes.ts',
            'apps/**/codegen.*',
            'libs/**/codegen.*',
            'apps/**/*.model.ts',
            'libs/**/*.model.ts',
            'apps/**/*.enum.ts',
            'libs/**/*.enum.ts',
            'apps/**/queries/**/*.tsx?',
            'libs/**/queries/**/*.tsx?',
            'libs/**/mutations/**/*.tsx?',
            'libs/**/fragments/**/*.tsx?',
            'apps/**/*.resolver.ts',
            'libs/**/*.resolver.ts',
            'apps/**/*.service.ts',
            'libs/**/*.service.ts',
            'apps/**/*.dto.ts',
            'libs/**/*.dto.ts',
            'apps/**/*.input.ts',
            'libs/**/*.input.ts',
            'apps/**/*.module.ts',
            'libs/**/*.module.ts',
            'apps/**/*.controller.ts',
            'libs/**/*.controller.ts',
            'apps/**/*.union.ts',
            'libs/**/*.union.ts',
            'apps/**/*.graphql.ts',
            'apps/judicial-system/**/*.graphql',
            'libs/**/*.graphql.ts',
            'libs/**/*.graphql',
            'libs/**/clientConfig.*'
          ) }}